src/main/java/com/actiontech/dble/route/sequence/handler/IncrSequenceZKHandler.java

```diff
@@ -135,14 +141,20 @@ private void handle(String key) throws Exception {
 
     @Override
     public Object[] getParaValMap(String prefixName) {
+        if (this.removeThreadList.remove(Thread.currentThread())) {
+            this.interProcessSemaphoreMutexThreadLocal.remove();
+            this.tableParaValMapThreadLocal.remove();
+        }
+        if (props.entrySet().isEmpty()) return null;
         Map<String, Map<String, String>> tableParaValMap = tableParaValMapThreadLocal.get();
         if (tableParaValMap == null) {
             try {
                 threadLocalLoad();
             } catch (Exception e) {
-                LOGGER.warn("Error caught while loding configuration within current thread:" + e.getCause());
+                LOGGER.warn("Error caught while loading configuration within current thread:" + e.getCause());
             }
             tableParaValMap = tableParaValMapThreadLocal.get();
+            threadList.add(Thread.currentThread());
```

Comment:
Need to consider deleting the collection data

Reply:
get